.github/workflows/test-windows.yml

```diff
@@ -69,22 +69,11 @@ jobs:
     - name: Print build system information
       run: python3 .github/workflows/system-info.py
 
-    - name: Install Python dependencies
-      run: >
-        python3 -m pip install
-        coverage>=7.4.2
-        defusedxml
-        olefile
-        pyroma
-        pytest
-        pytest-cov
-        pytest-timeout
-
     - name: Install CPython dependencies
       if: "!contains(matrix.python-version, 'pypy')"
-      run: >
-        python3 -m pip install
-        PyQt6
+      run: |
+        python3 -m pip install --upgrade pip
```

Comment:
This is not run on PyPy, only CPython. Maybe that is causing the issue?

Reply:
Let's see! It makes sense to upgrade for both anyway.

Reply:
Well, passing now, thanks :)